### PR TITLE
[docs] Updates to Readme

### PR DESCRIPTION
**Pull request by bsieber-mozilla**
Any other spots in the docs need updating @mozilla/jbi-reviewers ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ A bit more about the different fields...
     - Please enter a description; for example, team name or project use-case.
 - `enabled` (optional)
     - bool [true, false]
-    - default: false
+    - default: true
```
Comment (bsieber-mozilla) [inline README.md:48]: https://github.com/mozilla/jira-bugzilla-integration/pull/359

File: README.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ to the Bugzilla ticket on the Jira issue.
     - Whether to sync the Bugzilla status whiteboard labels to Jira. Defaults to `true`.
 - `status_map` (optional)
     - mapping [str, str]
-    - If defined, map the Bugzilla bug status to Jira issue status
+    - If defined, map the Bugzilla bug status (or resolution) to Jira issue status
```
Comment (bsieber-mozilla) [inline README.md:93]: https://github.com/mozilla/jira-bugzilla-integration/blob/f0922eda64bb1c83a6bb1bde2138973c89bce59b/jbi/actions/steps.py#L171-L178

**Review by leplatrem — APPROVED** [review]
This looks sound.

There are other bits of the README that can be slightly inaccurate like:
-  `The system sets the `see_also` field of the Bugzilla bug with the URL to the Jira issue`: this is now a possible step instead of a systematic behavior
- some missing comma in the `allow_private` parameter description
- In the `**Parameters**` section, the default values are not specified consistently
- Missing new line before `### Available Steps`

**Comment by bsieber-mozilla** [comment]
@leplatrem if the setting of the see_also is optional, how are we planning to "link" bugs instead of making dupes each time?

**Comment by bsieber-mozilla** [comment]
Merging this in; migrating other concerns to an issue to further discuss

**Pull request merged**